scripts/HPC_chatbot_preprocessor/parsed_mds/generic/FAQ/FAQ_paragraph_1.txt

```diff
@@ -0,0 +1,33 @@
+Frequently Asked Questions (FAQ)
+New users should consult the Introduction to HPC
```

Comment:
or move this intro ina FAQ of it's own: `Where to start when using the HPC` or something more proper english